patchwork/migrations/0047_timestampmixin_note.py

```diff
@@ -0,0 +1,73 @@
+# Generated by Django 5.0.3 on 2024-04-09 05:46
+
+import django.db.models.deletion
+import django.utils.timezone
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('patchwork', '0046_patch_comment_events'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='TimestampMixin',
+            fields=[
+                (
+                    'id',
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name='ID',
+                    ),
+                ),
+                (
+                    'created_at',
+                    models.DateTimeField(default=django.utils.timezone.now),
+                ),
+                (
+                    'updated_at',
+                    models.DateTimeField(default=django.utils.timezone.now),
+                ),
+            ],
+        ),
```

Comment:
This shouldn't be here. I'm going to guess you missed setting `abstract` in the `TimestampMixin.Meta` class?

Reply:
Added this setting